web/src/main/java/com/epam/esm/handler/GlobalControllerExceptionHandler.java

```diff
@@ -34,6 +38,17 @@ public CustomErrorResponse handleMediaTypeNotSupportedException(HttpMediaTypeNot
         return error;
     }
 
+    @ResponseStatus(HttpStatus.BAD_REQUEST)
+    @ExceptionHandler(MappingException.class)
+    public CustomErrorResponse handleMappingException(MappingException ex) {
```

Comment:
CustomErrorResponse is a bad naming(also as MyError and etc.) Please use more concrete error naming